hathor/indexes/manager.py

```diff
@@ -63,6 +64,22 @@ def enable_tokens_index(self) -> None:
         """Enable tokens index. It does nothing if it has already been enabled."""
         raise NotImplementedError
 
+    def _manually_initialize(self, tx_storage: 'TransactionStorage') -> None:
```

Comment:
I saw that we have a `BaseTransactionStorage._manually_initialize` method.

Does it make sense to make it call this new method?

Reply:
Yes, but not right now. That will make sense when we actually want to use this initialization. This PR aims to create and tests the new initialization function, but not enable it, because that will require more changes.